src/cli.cpp

```diff
@@ -56,7 +56,7 @@ string Strip0x(const string &hex)
 void HelpAndQuit(cxxopts::Options options)
 {
     cout << options.help({""}) << endl;
-    cout << "./ProofOfSpace generate" << endl;
+    cout << "./ProofOfSpace create" << endl;
```

Comment:
I assume this is related to a change in the underlying Proofofspace script? Any other impact here other than a rename of the function?

Reply:
This builds a c++ executable called ProofOfSpace and a python binary wheel. In chia-blockchain the app calls it `chia plots create` so when you switch to use the executable it's super easy to do this `./ProofOfSpace -k 32 create &`. Now that will work. Nothing depends on the ProofOfSpace command line arguments beyond itself.